django/contrib/admin/templates/admin/base.html

```diff
@@ -10,6 +10,9 @@
 {% block extrastyle %}{% endblock %}
 {% if LANGUAGE_BIDI %}<link rel="stylesheet" type="text/css" href="{% block stylesheet_rtl %}{% static "admin/css/rtl.css" %}{% endblock %}">{% endif %}
 {% block extrahead %}{% endblock %}
+{% if not is_popup and is_nav_sidebar_enabled %}
```

Comment:
Would it make sense to reused the `{% if %}` on line 7?

Reply:
It doesn't work for me when it's on line 7 :thinking: